### ISSUE TITLE
YOU ARE A FUCKING LEGEND

### ISSUE THREAD

**Issue by redbastie** [1kc/razer-macos#90]

**Comment by 1kc** [comment]
🙏🙏

On 5 Nov 2020, at 9:04 am, redbastie <notifications@github.com> wrote:

﻿

**Comment by twistedpixel** [comment]
I completely agree. This project is phenomenal: you've done an incredible job supporting the Mac in a way Razer haven't bothered with for years.

I'd like to volunteer to help out however I can with this project and in particular, I'd like to help move the app forward in terms of more general functionality such as button bindings, DPI changes, etc. (although I see someone has already submitted a pull request for DPI changes?)

FYI I have a Razer Basilisk Ultimate which would be more or less useless on my Macs without your app. I'm also looking to get a Razer keyboard so can probably help with testing there too.

I've been meaning to learn how to use Electron for a while now and this might be the ideal project to start with!

**Comment by 1kc** [comment]
@twistedpixel thanks for your nice words! Please give the new release with DPI changing a go. Highly encourage you to have a play around with Electron (and React) if you've got the time. That's how I learnt it.

**Comment by twistedpixel** [comment]
> @twistedpixel thanks for your nice words! Please give the new release with DPI changing a go. Highly encourage you to have a play around with Electron (and React) if you've got the time. That's how I learnt it.

Just tested the new release: DPI functionality working excellent on my Basilisk Ultimate 👍 super helpful too as the sensitivity buttons on this mouse are useless on Mac by default!

**Comment by 1kc** [comment]
@twistedpixel You should thank absolute legend @GBrawl for that 😂!
